travis-ci/vmtest/run.sh

```diff
@@ -385,16 +386,23 @@ if (( SKIPSOURCE )); then
 	echo "Not copying source files..." >&2
 else
 	echo "Copying source files..." >&2
-
 	# Copy the source files in.
 	sudo mkdir -p -m 0755 "$mnt/${PROJECT_NAME}"
-	{
-	if [[ -e .git ]]; then
-		git ls-files -z
+	if [ "${SOURCE_FULLCOPY:-1}" == "1" ]; then
```

Comment:
where is this ${SOURCE_FULLCOPY} set? It might be good to put this to the top if the file with some document on what this variable does.